bbb-lti/gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
I notice a file permission change 644 -> 755. Is this intentional?

Reply:
I'm not sure about it because the file is auto-generated by gradle. This might has happened due to change in gradle version?